src/urllib3/util/ssl_.py

```diff
@@ -349,14 +348,9 @@ def ssl_wrap_socket(
     if ca_certs or ca_cert_dir or ca_cert_data:
         try:
             context.load_verify_locations(ca_certs, ca_cert_dir, ca_cert_data)
-        except IOError as e:  # Platform-specific: Python 2.7
+        except IOError as e:
```

Comment:
I'm not sure if we want to be more strict in Python 3 here?

Also should we keep `except (IOError, OSError) as e` just to enhance confidence in this change even if it's a no-op?

Reply:
I meant, we could catch a FileNotFoundErro in Python3, but I removed that for now.